distribution/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:6.10
+RUN yum install --assumeyes gcc
```

Comment:
there's a funny bit in the latest season of arrested development that this reminds me of

![presume teeth](https://preview.redd.it/e7ajla4zf9q21.png?width=757&auto=webp&s=96fdf52f957f9344015350bb03a7d9304d3bc346)